test/io/testfilereader.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Unit tests for general file reader."""
+
+import inspect
+import os
+import unittest
+
+import cclib
+
+
+__filedir__ = os.path.dirname(__file__)
+__filepath__ = os.path.realpath(__filedir__)
+__datadir__ = os.path.join(__filepath__, "..", "..")
+
+
+class FileReaderTest(unittest.TestCase):
+
+    def test_type(self):
+        """Is the class properly a metaclass?"""
+
+        self.assertTrue(inspect.isabstract(cclib.io.filereader.Reader))
```

Comment:
We can leave this, but I don't see much value in testing the implementation of Python. Is there something meaningful about the base class that we can test?